design/high-level-reactivity.md

```diff
@@ -205,10 +248,19 @@ context.invalidateDependency(notificationDepKeyForUser(userId));
 
 Hopefully, with time, we can make more and more invalidations automatic, but it’s always good to have an “escape hatch” for more complex situations where the developer needs all the control they can get.
 
+// Add a section here: This is why we said, you need to add custom keys. 
+// Then we can have some examples like this:
+
+1. Stop reactivity. Just publish a depKey which is random. Do not invalidate that key.
+2. Counter.  Sometimes you don't need to it to be neal realtime. So, we can throttle the invalidations and invalidate it for every few minutes.
+
 <img src="reactive-with-mutations-diagram.png" title="Reactive GraphQL with mutations" />
 
 You can see in the diagram how invalidations flow from the mutation to the relevant clients, which then refetch the data as needed.
 
+// How about outside invalidations. Lets say after a diret DB write from another non Meteor app.  
+// If we can connect to the invalidation server from outside that's awesome. (Again via a GraphQL mutation)
```

Comment:
Of course! The invalidation component must have a clear API that anyone can call, so you can push changes from any microservice. This is crucial for all kinds of applications, like chat services. I'm excited to build a prototype to see how well this will work.